lib/Sema/TypeCheckConcurrency.cpp

```diff
@@ -4782,20 +4795,27 @@ VarDecl *swift::getReferencedParamOrCapture(
 }
 
 bool swift::isPotentiallyIsolatedActor(
-    VarDecl *var, llvm::function_ref<bool(ParamDecl *)> isIsolated) {
+    VarDecl *var,
+    llvm::function_ref<bool(ParamDecl *)> isIsolated,
+    llvm::function_ref<bool(ParamDecl *)> isDistributedKnownLocal) {
   if (!var)
     return false;
 
-  if (var->getName().str().equals("__secretlyKnownToBeLocal")) {
-    // FIXME(distributed): we did a dynamic check and know that this actor is
-    //   local, but we can't express that to the type system; the real
-    //   implementation will have to mark 'self' as "known to be local" after
-    //   an is-local check.
-    return true;
-  }
+//  if (var->getName().str().equals("__secretlyKnownToBeLocal")) {
```

Comment:
I suspect you'll need to keep this hack around for a little while.

Reply:
Okey will do 👍

Reply:
keeping it